governance/core-team/meeting-notes/20230928-core-team.md

```diff
@@ -0,0 +1,283 @@
+# *The Turing Way* Community Call - 28 September 2023
+
+*This document is for the Community Call on Thursday 28 September 2023.*
+
+Join Zoom Meeting
+
+[https://turing-uk.zoom.us/j/93568219227?pwd=MEZaSjJzUDJqb2RtbjZCTDh2eklidz09](https://turing-uk.zoom.us/j/93568219227?pwd=MEZaSjJzUDJqb2RtbjZCTDh2eklidz09)
+
+Meeting ID: 93568219227
+
+Passcode: 600713
+
+
+**Quick reminders**
+
+   * *The Turing Way* participation guidelines apply to this event:
+       * Code of Conduct: [https://the-turing-way.netlify.app/community-handbook/coc.html](https://the-turing-way.netlify.app/community-handbook/coc.html)
+       * Contribution guideline: [https://github.com/alan-turing-institute/the-turing-way/blob/master/CONTRIBUTING.md](https://github.com/alan-turing-institute/the-turing-way/blob/master/CONTRIBUTING.md)
+   * Time is provided in London Time (UTC+1). Please use this link to convert in your time zone: convert time: [https://arewemeetingyet.com/London/2023-09-28/15:30/CommunityCall](https://arewemeetingyet.com/London/2023-09-28/15:30/CommunityCall)
+   * For any question or concern please reach out to The Turing Way team by emailing turingway@turing.ac.uk, or individually reach out to Malvika, Alexandra or Anne (msharan@turing.ac.uk, asteele@turing.ac.uk, aaraujo.alvarez@turing.ac.uk).
+
+
+*Name / Organisation / Turing Way area of focus (if applicable)*
+
+   * Arielle Bennett / The Alan Turing Institute / Research Infrastructure Roles, Activism, Book Dash
```

Comment:
I.e. maybe we can anonymise these names? Will investigate and add more information here before we merge.